### PR TITLE
Update AE table templates with modifications from chevron

### LINKED ISSUE #59: Update AE table templates with modifications from chevron
See https://github.com/insightsengineering/sme-tasks/issues/617.

Templates to update:

- [x]  AET01
- [x] AET01_AESI
- [x]  AET02
- [x]  AET03
- [x]  AET04

### PR DESCRIPTION
**Pull request by edelarua**
Closes #59 and #55

Also fixes indentation in RMPT06 and TTET01 and simplifies `count_patients_with_flags` usage in AE/PK templates.

See https://github.com/insightsengineering/scda.test/pull/55 for corresponding snapshot updates of these templates.

### CODE REVIEW

File: book/tables/adverse-events/aet01_aesi.qmd [hunk 1]
```diff
@@ -85,7 +85,7 @@ adae <- adae %>%
 
 ## Standard Table
 
-```{r}
```
Comment (Melkiades) [inline book/tables/adverse-events/aet01_aesi.qmd:108]: Nice catch!

File: book/_quarto.yml [hunk 2]
```diff
@@ -44,7 +44,7 @@ knitr:
     echo: true
     fig.width: 15
 
-editor: visual
```
Comment (Melkiades) [inline book/_quarto.yml:47]: where is this coming from?

Comment (edelarua) [reply book/_quarto.yml:47]: Oh sorry, I'll revert that. It just stops the qmd docs from automatically opening in visual editor mode - I prefer the other mode for development so use this on my local.

**Review by Melkiades — APPROVED** [review]
Thanks Emily for this! :) lgtm!

**Pull request merged**